### PR TITLE
ttlafterfinish to beta

### PR DESCRIPTION
**Pull request by ahg-g**
Ref https://github.com/kubernetes/enhancements/issues/592

### CODE REVIEW

**Review by janetkuo — APPROVED** [review]

File: content/en/docs/concepts/workloads/controllers/ttlafterfinished.md [hunk 1]
```diff
@@ -8,17 +8,17 @@ weight: 70
...
+{{< feature-state for_k8s_version="v1.21" state="beta" >}}
 
 The TTL controller provides a TTL (time to live) mechanism to limit the lifetime of resource
 objects that have finished execution. TTL controller only handles
 {{< glossary_tooltip text="Jobs" term_id="job" >}} for now,
 and may be expanded to handle other resources that will finish execution,
 such as Pods and custom resources.
 
-Alpha Disclaimer: this feature is currently alpha, and can be enabled with both kube-apiserver and kube-controller-manager
+This feature is currently beta and enabled by default, and can be disabled via
```
Comment (sftim) [inline content/en/docs/concepts/workloads/controllers/ttlafterfinished.md:19]: nits (these don't block a merge):
```suggestion
This feature is beta and enabled by default. You can disable it using the
```

**Comment by sftim** [comment]
Given that https://github.com/kubernetes/kubernetes/pull/98678 is merged:
/approve

**Pull request merged**